client/wp-admin-scripts/print-shipping-label-banner/shipping-banner/index.js

```diff
@@ -349,7 +350,7 @@ export class ShippingBanner extends Component {
 					</h3>
 					<p>
 						{ interpolateComponents( {
-							mixedString: this.getInstallText(),
+							mixedString: this.state.installText,
```

Comment:
Since `getInstallText()` function return text based on the `activePlugins` props, the text will get updated when `wc-api` re-render by updating `activePlugins` prop.

I could add logics with this state `isShippingLabelButtonBusy` inside `getInstallText()` , so that it will return different text depending on if the "Create shipping label" has been clicked or not.

Instead, I decided to add a local state `getInstallText` that is initialized once at constructor. This state will not be updated during re-render so our message will not change throughout.